cg/services/quality_controller/quality_checks/utils.py

```diff
@@ -0,0 +1,132 @@
+import logging
+
+from cg.constants.constants import PrepCategory
+from cg.constants.priority import Priority
+from cg.store.models import Case, Sample
+
+LOG = logging.getLogger(__name__)
+
+
+def case_pass_sequencing_qc(case: Case) -> bool:
+    """
+    Get the standard sequencing QC of a case. If the case is express priority, the express QC is
+    used. If the case is a ready made library, the ready made library QC is used.
+    """
+    if is_case_ready_made_library(case):
+        return ready_made_library_case_pass_sequencing_qc(case)
+    if is_case_express_priority(case):
+        return express_case_pass_sequencing_qc(case)
```

Comment:
Is the order relevant here, checking first `is_case_ready_made_library` and then `is_case_express_priority`? Maybe we should be more explicit in the docstrings about it.

Reply:
yeah it is. I would like to change this, I dont like that the order of it matters. I will think about it!